modules/bitgo/test/v2/unit/coins/ofcToken.ts

```diff
@@ -23,6 +23,13 @@ describe('OFC:', function () {
     otestusdCoin.getBaseFactor().should.equal(PRECISION_2);
   });
 
+  it('functions that return constants', function () {
```

Comment:
Should we change this to something different than `functions that return constants` as it was used for the ofctusd test above?